stream_alert_cli/config.py

```diff
@@ -116,26 +70,29 @@ def set_prefix(self, prefix):
             LOGGER_CLI.error('Prefix cannot contain underscores')
             return
 
+        tf_state_bucket = '{}.streamalert.terraform.state'.format(prefix)
```

Comment:
(not related to this line)

Any reason to delete the `generate_athena` method? It could still be useful for current users who don't have an Athena config

Reply:
We don't auto-generate any other part of the config (e.g. Lambda template code for the alert processor or alert merger), and I think we should be consistent. So either we should auto-generate all of it or none of it, and IMO it would be over-engineering to auto-generate every part of the config an upgrading user might not have (because they'll likely need to change the config anyway). With documentation, users can just update their config manually. So my vote is to get rid of the `generate_athena` method

#simplify 💣

Reply:
@austinbyers and @jacknagz maybe we need to discuss this because I'm not sure what the best approach is. my initial impression was to nix it as well (as seen in my commit history). let's chat today